src/main/kotlin/com/colivery/serviceaping/rest/v1/responses/UserOrderSearchResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.colivery.serviceaping.rest.v1.responses
+
+import com.colivery.serviceaping.rest.v1.resources.AnonymizedUserResource
+import com.colivery.serviceaping.rest.v1.resources.OrderResource
+
+data class UserOrderSearchResponse(
+        val order: OrderResource,
+        val user: AnonymizedUserResource?
```

Comment:
I think there should always be a User in the response, or am I missing something?

Reply:
You are right